site/content/protocols/mediator-coordination/3.0/readme.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Mediator Coordinator
+publisher: rodolfomiranda
+license: MIT
+piuri: https://didcomm.org/coordinate-mediation/3.0
+status: Production
+summary: A protocol to coordinate mediation configuration between a mediating agent and the recipient.
+tags: []
+authors:
```

Comment:
Should rodolfo and you be added as authors here?

Reply:
I could certainly add those. I didn't touch the authors field because I didn't think that renaming things (with no functionality changes otherwise) warranted being an author.